### ISSUE TITLE
[UI Bug] Cavern Crawl menu The Candyworks button overlaps cavern crawl nodes making them un-interactable

### ISSUE THREAD

**Issue by allengar22** [ValveSoftware/Dota2-Gameplay#4864]
### Description

You cannot hover or select this node at the edge of the screen meaning you can't see what heroes you need to play or use items on it

### Screenshots

![image](https://user-images.githubusercontent.com/9031705/200091025-e332efce-1e94-4b56-bd1e-6e7183f71dcf.png)

**Comment by commentator24** [comment]
hmm footless carry.
i can only think of sladar or viper. what is it? XD

**Comment by Toorero** [comment]
> hmm footless carry. i can only think of sladar or viper. what is it? XD

How is this adding anything to the bug-report?

**Comment by Toorero** [comment]
> You cannot hover or select **this node at the edge of the screen** meaning you can't see what heroes you need to play or use items on it

This is true for me as well for quests overlapping with the arrows to switch perspective, basically any UI not only the edge of the screen.

**Comment by Toorero** [comment]
#4759
